packages/flame/lib/src/extensions/matrix4.dart

```diff
@@ -78,6 +80,13 @@ extension Matrix4Extension on Matrix4 {
     return transform2(out);
   }
 
+  /// Transform [rect] of type [Rect] using the transformation defined by this.
```

Comment:
This method works only with translation transforms and non-negative scale transforms, right?

Reply:
Correct, should add that to the dartdoc

Reply:
If you have any suggestions, I am all ears.